react.js

```diff
@@ -0,0 +1,71 @@
+module.exports = {
+	extends: ["airbnb", "airbnb/hooks"],
+	plugins: [],
+	env: {
+		es6: true,
+		node: true,
+		browser: true,
+	},
+	parserOptions: {
+		ecmaVersion: 6,
+		ecmaFeatures: {
+			jsx: true,
+		},
+		sourceType: "script",
+	},
+	rules: {
+		indent: [2, "tab"],
+		"prettier/prettier": "error",
```

Comment:
What do you think of using prettier in Kink-Web?

Reply:
I would love to do it, I dont know why I didnt add it from the start